src/components/CategoryList/CategoryList.js

```diff
@@ -0,0 +1,28 @@
+import Grid from '@mui/material/Unstable_Grid2';
+
+import CategoryButton from '../../components/CategoryButton/CategoryButton';
+
+const CategoryList = (props) => {
+  const {
+    rowSpacing,
+    columnSpacing,
+    xsColumns,
+    smColumns,
+    mdColumns,
+    xlColumns,
+    array,
+    onClick,
+  } = props;
+
+  return (
+    <Grid container rowSpacing={rowSpacing} columnSpacing={columnSpacing}>
+      {Array.from(array).map((el, index) => (
+        <Grid key={index} xs={xsColumns} sm={smColumns} md={mdColumns} xl={xlColumns}>
```

Comment:
If you have a unique value in the array, put it into the key parameter and avoid using the index. 
```suggestion
        <Grid key={el.name} xs={xsColumns} sm={smColumns} md={mdColumns} xl={xlColumns}>
```